source/kraken/maintenance_worker.cpp

```diff
@@ -145,7 +144,8 @@ void MaintenanceWorker::handle_task_in_batch(const std::vector<AmqpClient::Envel
         }
         switch(task.action()){
             case pbnavitia::RELOAD:
-                load();
+                this->load_data();
+                this->load_realtime();
```

Comment:
Not sure, this may fail but it's still unprotected, no?

Reply:
errors are handled here https://github.com/benoit-bst/navitia/blob/ade56a81277fc83d536b9215cc3607510382d44e/source/kraken/maintenance_worker.cpp#L128-L132